src/material-experimental/mdc-checkbox/checkbox.ts

```diff
@@ -156,6 +158,7 @@ export class MatCheckbox implements AfterViewInit, OnDestroy, ControlValueAccess
   }
   set required(required) {
     this._required = coerceBooleanProperty(required);
+    this._changeDetectorRef.markForCheck();
```

Comment:
Not entirely sure what we want to do in general about these. I added a note for this to the meeting so we can chat about it.

Reply:
I think this makes sense, @crisbeto mentioned a while ago that I should add them, but I forgot